2020-04-06 00:22:57.994 [error] <0.367.0>@Elixir.IO:inspect CRASH REPORT Process <0.367.0> with 0 neighbours crashed with reason: call to undefined function 'Elixir.IO':inspect()
2020-04-06 00:24:45.707 [error] <0.362.0> Supervisor 'Elixir.Frequency.Supervisor' had child 'Elixir.Frequency.Worker' started with 'Elixir.Frequency.Worker':start_link([]) at undefined exit with reason {'EXIT',{undef,[{'Elixir.Frequency.Worker',start_link,[[]],[]},{supervisor,do_start_child_i,3,[{file,"supervisor.erl"},{line,379}]},{supervisor,do_start_child,2,[{file,"supervisor.erl"},{line,365}]},{supervisor,'-start_children/2-fun-0-',3,[{file,"supervisor.erl"},{line,349}]},{supervisor,children_map,4,[{file,"supervisor.erl"},{line,1157}]},{supervisor,init_children,2,[{file,"supervisor.erl"},{line,315}]},{gen_server,init_it,2,[{file,"gen_server.erl"},{line,374}]},{gen_server,init_it,6,[{file,...},...]},...]}} in context start_error
2020-04-06 00:24:58.442 [error] <0.362.0> Supervisor 'Elixir.Frequency.Supervisor' had child 'Elixir.Frequency.Worker' started with 'Elixir.Frequency.Worker':start_link([]) at undefined exit with reason {'EXIT',{undef,[{'Elixir.Frequency.Worker',start_link,[[]],[]},{supervisor,do_start_child_i,3,[{file,"supervisor.erl"},{line,379}]},{supervisor,do_start_child,2,[{file,"supervisor.erl"},{line,365}]},{supervisor,'-start_children/2-fun-0-',3,[{file,"supervisor.erl"},{line,349}]},{supervisor,children_map,4,[{file,"supervisor.erl"},{line,1157}]},{supervisor,init_children,2,[{file,"supervisor.erl"},{line,315}]},{gen_server,init_it,2,[{file,"gen_server.erl"},{line,374}]},{gen_server,init_it,6,[{file,...},...]},...]}} in context start_error
2020-04-06 00:25:56.953 [error] <0.362.0> Supervisor 'Elixir.Frequency.Supervisor' had child 'Elixir.Frequency.Worker' started with 'Elixir.Frequency.Worker':start_link([]) at undefined exit with reason {'EXIT',{undef,[{'Elixir.Frequency.Worker',start_link,[[]],[]},{supervisor,do_start_child_i,3,[{file,"supervisor.erl"},{line,379}]},{supervisor,do_start_child,2,[{file,"supervisor.erl"},{line,365}]},{supervisor,'-start_children/2-fun-0-',3,[{file,"supervisor.erl"},{line,349}]},{supervisor,children_map,4,[{file,"supervisor.erl"},{line,1157}]},{supervisor,init_children,2,[{file,"supervisor.erl"},{line,315}]},{gen_server,init_it,2,[{file,"gen_server.erl"},{line,374}]},{gen_server,init_it,6,[{file,...},...]},...]}} in context start_error
2020-04-06 00:26:32.405 [error] <0.362.0> Supervisor 'Elixir.Frequency.Supervisor' had child 'Elixir.Frequency.Worker' started with 'Elixir.Frequency.Worker':start_link([]) at undefined exit with reason {'EXIT',{undef,[{'Elixir.Frequency.Worker',start_link,[[]],[]},{supervisor,do_start_child_i,3,[{file,"supervisor.erl"},{line,379}]},{supervisor,do_start_child,2,[{file,"supervisor.erl"},{line,365}]},{supervisor,'-start_children/2-fun-0-',3,[{file,"supervisor.erl"},{line,349}]},{supervisor,children_map,4,[{file,"supervisor.erl"},{line,1157}]},{supervisor,init_children,2,[{file,"supervisor.erl"},{line,315}]},{gen_server,init_it,2,[{file,"gen_server.erl"},{line,374}]},{gen_server,init_it,6,[{file,...},...]},...]}} in context start_error
